### PR TITLE
Bloodhound 0.13

### LINKED ISSUE #30: Support bloodhound 0.13 in log-elasticsearch
There's a new `bloodhound` release out, which splits the module hierarchy in `V1` and `V5` namespaces. We should add support for this to `log-elasticsearch`, probably by doing a similar split in our module structure.

### PR DESCRIPTION
**Pull request by phadej**
Resolve #30 

I went with dumb approach, as clever type-family stuff made code too complicated. With a copying script, keeping two versions compatible should be simple

### CODE REVIEW

**Comment by phadej** [comment]
I have to update `log`, and will do if this approach is ok.

**Comment by 23Skidoo** [comment]
I'm on vacation, will review in a couple of days.

**Comment by phadej** [comment]
`bloodhound-0.14.0.0` is released, should I bump the lower-bound to it?

**Comment by 23Skidoo** [comment]
Yeah, sure. Sorry for being a bit slow on this, will review tomorrow.

**Comment by 23Skidoo** [comment]
I'm OK with the dumb approach for now, though once the two files will begin to diverge, it'll not be fun to maintain. We can drop V1 API at that point, but as long as it's not too onerous, let's continue supporting it.

File: log-elasticsearch/log-elasticsearch.cabal [hunk 1]
```diff
@@ -24,16 +24,21 @@ Source-repository head
...
 library
-  exposed-modules:     Log.Backend.ElasticSearch
-                       Log.Backend.ElasticSearch.Internal
+  exposed-modules:     Log.Backend.ElasticSearch.V1
+                       Log.Backend.ElasticSearch.V1.Internal
+                       Log.Backend.ElasticSearch.V1.Lens
+                       Log.Backend.ElasticSearch.V5
+                       Log.Backend.ElasticSearch.V5.Internal
+                       Log.Backend.ElasticSearch.V5.Lens
+                       Log.Backend.ElasticSearch
```
Comment (23Skidoo) [inline log-elasticsearch/log-elasticsearch.cabal]: If the `L.B.ES.Internal` module is now gone, we'll have to do a major version bump. Probably safer to do one anyway.

**Comment by phadej** [comment]
ping @23Skidoo, the build is green

File: log-elasticsearch/log-elasticsearch.cabal (hunk 1 above)
Comment (phadej) [inline log-elasticsearch/log-elasticsearch.cabal]: I'm not sure, adding one would be safer indeed. Also I'll add deprecated shim modules to `log` too.

**Comment by 23Skidoo** [comment]
Merged, thanks!

**Pull request merged**